### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by pgmillon**
This change is [Reviewable](https://reviewable.io/reviews/nuxeo-sandbox/nuxeo-pullrequest/1)

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/nuxeo-sandbox/nuxeo-pullrequest/pull/1" target="_blank">ReviewNinja

**Comment by jcarsique** [comment]
why not

---

Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---

_[README.md, line 2 [r1]](https://reviewable.io:443/reviews/nuxeo-sandbox/nuxeo-pullrequest/1#-KE_n_vQgfH-M1epr7Eh:-KE_n_vQgfH-M1epr7Ei:-326875929) ([raw file](https://github.com/nuxeo-sandbox/nuxeo-pullrequest/blob/e8b6df75284bdf5077acb50a7f6db7be2bc9ca4e/README.md#L2)):_
remove WIP

---

_[README.md, line 4 [r1]](https://reviewable.io:443/reviews/nuxeo-sandbox/nuxeo-pullrequest/1#-KE_nGt0umXDPuRmizc_:-KE_nGt0umXDPuRmizca:-617500014) ([raw file](https://github.com/nuxeo-sandbox/nuxeo-pullrequest/blob/e8b6df75284bdf5077acb50a7f6db7be2bc9ca4e/README.md#L4)):_
Not the right text

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/nuxeo-sandbox/nuxeo-pullrequest/1#-:-KE_oHbWXwo1plp7oUnM:-817165866)_

File: README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# nuxeo-pullrequest
+Nuxeo Pull Request WIP
```
Comment (jcarsique) [inline README.md:2]: remove WIP !resolve

Comment (jcarsique) [inline README.md:2]: ok !resolved

Comment (jcarsique) [inline README.md:2]: !completed

**Comment by pgmillon** [comment]
:+1:

**Pull request merged**